app/templates/model-details.html

```diff
@@ -108,31 +111,43 @@ <h2 class="panel-title">{{ getActiveModelData('name') }}</h2>
                     :disabled="(isReadOnly || !isIdle)">
               <span class="btn-label"><i class="fa fa-fw fa-play" aria-hidden="true"></i></span>
               Start
-            </button>
+            </button>&nbsp;
```

Comment:
These are for better styling: the buttons are all inline, but the jquery tooltip removes any spaces between buttons which are on a new line in the html - the &nbsp; prevents that from happening.